mozilla-release/mobile/android/base/java/org/mozilla/gecko/activitystream/homepanel/ActivityStreamPanel.java

```diff
@@ -140,10 +147,23 @@ public ActivityStreamPanel(final Context context, AttributeSet attrs) {
         customizeNewTabViewSnackBar = findViewById(R.id.customize_newtab_snackbar);
 
         final View customizeTabLink = customizeNewTabView.findViewById(R.id.customize_tab_link);
+        if (abManager.getHomeSettingsStyle() == "text") {
+            customizeTabLink.setBackgroundColor(Color.argb(0, 0, 0, 0));
+        }
         customizeTabLink.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
                 final Intent intent = new Intent(context, GeckoPreferences.class);
+                final GeckoBundle signal = new GeckoBundle();
+                signal.putString("type", "onboarding");
```

Comment:
How many telemetry signal will have the same type? More than 1 -> we should generalize

Reply:
`sendTelemetry` accepts a Geckobundle.

We will have 2 signals, so I will generalize. Shall I create a GoSettingsTelemetry class? Or maybe a HomePageTelemetry? Where should I create this file?